src/test/java/net/datafaker/providers/base/FinanceTest.java

```diff
@@ -55,6 +56,20 @@ void ibanWithCountryCode() {
         assertThat(finance.iban("DE")).matches("DE\\d{20}");
     }
 
+    @Test
+    void ibanCountryCodes() {
+        assertThat(Finance.ibanSupportedCountries()).isNotEmpty().hasSizeGreaterThan(70);
+    }
+
+    @Test
+    public void ibanWithAllCountryCodes() {
```

Comment:
small nitpick: could you remove the public keyword?

Reply:
Done, haven't written Java for a while ;-)